runtime/common/src/lib.rs

```diff
@@ -70,26 +73,19 @@ pub mod opaque {
 	pub type BlockId = generic::BlockId<Block>;
 }
 
-pub struct RelativeValue<Amount> {
-	pub num: Amount,
-	pub denominator: Amount,
-}
-
-impl<Amount: CheckedDiv<Output = Amount> + Saturating + Clone> RelativeValue<Amount> {
-	pub fn divide_by_relative_value(
-		amount: Amount,
-		relative_value: RelativeValue<Amount>,
-	) -> Amount {
-		// Calculate the adjusted amount
-		if let Some(adjusted_amount) = amount
-			.clone()
-			.saturating_mul(relative_value.denominator)
-			.checked_div(&relative_value.num)
-		{
-			return adjusted_amount
-		}
-		// We should never specify a numerator of 0, but just to be safe
-		return amount
+pub struct FixedConversionRateProvider<OrmlAssetRegistry>(PhantomData<OrmlAssetRegistry>);
```

Comment:
Maybe it's better to have this in `asset_registry.rs` instead? WDYT?

Reply:
Yes I agree! moving.